### ISSUE TITLE
WebdriverIO v5.x Chrome capabilities

### ISSUE THREAD

**Issue by carlos-gva** [webdriverio/webdriverio#3238]
Correct me if I'm wrong, but now with the W3C standards we should define `chromeOptions` with the prefix `goog:`, right?

And what about `applicationName`? Did it disappear and now it's not possible to use ir for example to run different tests on different Chrome nodes on my Docker Selenium Grid?

**Comment by abjerstedt** [comment]
@carlos-gva Yes. chromeOptions is now goog:chromeOptions if you are recent versions of selenium-standalone-server and chromedriver.

I'm not sure about applicationName, but have you tried just specifying the devicename/id?

**Comment by carlos-gva** [comment]
@abjerstedt Thanks!
No, I didn't try that. And does not seem to be available on a Docker Chrome Node configuration. 🤔 
https://github.com/SeleniumHQ/docker-selenium/tree/master/NodeChrome

**Comment by carlos-gva** [comment]
@christian-bromann Sorry, my bad.
